dbcon/joblist/resourcemanager.h

```diff
@@ -82,6 +82,7 @@ const uint32_t defaultRequestSize  = 1;
 const uint32_t defaultMaxOutstandingRequests = 20;
 const uint32_t defaultProcessorThreadsPerScan = 16;
 const uint32_t defaultJoinerChunkSize = 16 * 1024 * 1024;
+const uint64_t defaultMaxBPPSendQueue = 250000000; // 250MB
```

Comment:
What is the reasoning behind this value? Could you share it?

Reply:
I tried various values on my 16GB VM. This appeared a good compromise. It may not be optimal. The thought process is fuzzy.

Reply:
In most cases I've seen non-cloud clients uses hardware that has more than 16GB RAM that is why I am skeptical on this value. 
Could you share the reasons you used to arrive at this compromise?

Reply:
I estimate that the average non-cloud columnstore node has 64GB. I've seen from 16GB to 256GB. Cloud can be as low as 4GB (some people are too cheap for their own good). 

However, ExeMgr has a targetRecvQueueSize hardcoded to 50,000,000, so some number greater than this makes sense. Seriously greater doesn't make sense, so I went with 5x. If there are a number of simultaneous queries that return giant result sets, then 0.25 GB each seems reasonable. This is only for the return queue. We still need room for all the processing, and if a single node system, for ExeMgr as well. On small systems, I recommend we use a smaller value. 

I believe a larger value will not improve anything since at this point, we're just filling a queue much faster than it can be emptied. Even if we make this default larger, giant results will still eventually block. Just with less memory available for other processing.